packages/neon/neon/lib/src/utils/global_options.dart

```diff
@@ -129,27 +140,41 @@ class GlobalOptions extends OptionsCollection {
     },
   );
 
+  /// Whether the [ThemeMode.dark] should use a plain black background.
+  ///
+  /// This is commonly used on oled devices.
+  /// Defaults to `false`.
   late final themeOLEDAsDark = ToggleOption(
     storage: storage,
     key: GlobalOptionKeys.themeOLEDAsDark,
     label: (final context) => NeonLocalizations.of(context).globalOptionsThemeOLEDAsDark,
     defaultValue: false,
   );
 
+  /// Whether the `ColorScheme` should keep the Nextcloud you provided color.
```

Comment:
```suggestion
  /// Whether the `ColorScheme` should keep the accent color provided by the Nextcloud server.
```
Talking of Nextcloud you is confusing and it is just a marketing term anyway.